test/fixtures/controller.line/borderWidth/scriptable.js

```diff
@@ -26,19 +24,20 @@ module.exports = {
 			title: false,
 			elements: {
 				line: {
-					fill: false,
-				},
-				point: {
 					borderColor: '#ff0000',
 					borderWidth: function(ctx) {
-						var value = ctx.dataset.data[ctx.dataIndex] || 0;
-						return value > 4 ? 2
-							: value > -4 ? 5
-							: 10;
+						var index = (ctx.dataIndex === undefined ? ctx.datasetIndex : ctx.dataIndex);
+						return index === 0 ? 20 : 10;
 					},
+					fill: false,
+				},
+				point: {
 					radius: 10,
```

Comment:
I think we should set a default point border color `#00ff00` (element) so it doesn't mix with the line border in the first dataset. It would also make clearer that in the second dataset, the point border isn't impacted by the default line options.